Pear/Controller/HomeViewController.swift

```diff
@@ -94,6 +94,30 @@ class HomeViewController: UIViewController {
         navigationController?.setNavigationBarHidden(false, animated: false)
     }
 
+    private func showInAppFeedback() {
+        guard let netId = UserDefaults.standard.string(forKey: Constants.UserDefaults.userNetId) else { return }
+        NetworkManager.shared.getMatchHistory(netID: netId).observe { response in
+            switch response {
+            case .value(let value):
+                guard value.success else {
+                    print("Network error: could not get user match history")
+                    return
+                }
+                var previousMatchHistorySize = UserDefaults.standard.integer(forKey: Constants.UserDefaults.previousMatchHistorySize)
+                previousMatchHistorySize = previousMatchHistorySize == 0 ? 1 : previousMatchHistorySize
+                if (value.data.count > previousMatchHistorySize) {
+                    DispatchQueue.main.async {
+                        let navController = UINavigationController(rootViewController: FeedbackViewController())
```

Comment:
I don't know, but you should double check that you want to present another navigation controller over whatever's already there. Maybe this makes sense, but let's be certain.